soepy/test/random_init.py

```diff
@@ -217,7 +206,25 @@ def random_init(constr=None):
 
     random_model_params_df.to_pickle("test.soepy.pkl")
 
-    return model_spec_init_dict, random_model_params_df
+    # Generate random probabilities of childbirth
+    exog_child_info_random = np.random.uniform(0, 1, size=periods).tolist()
+    exog_child_info_random_dict = {"prob_child_values": exog_child_info_random}
```

Comment:
This all can be done much simpler. Let's discuss during the call.

Reply:
Done.